assets/src/edit-story/components/canvas/singleSelectionMovable.js

```diff
@@ -123,15 +123,17 @@ function SingleSelectionMovable( {
 	const isTextElement = 'text' === selectedElement.type;
 	const shouldAdjustFontSize = isTextElement && selectedElement.content.length && isResizingFromCorner;
 
+	const isMedia = 'image' === selectedElement.type || 'video' === selectedElement.type;
```

Comment:
WDYT if we moved it to `element/x/index.js`. E.g.:

```
const { keepRatio } = getDefinitionForType( selectedElement.type );
```

Thoughts?

Reply:
Good idea. `keepRatio` might not be entirely accurate here though since it keeps ratio only when resizing from corners.

Will add `{ isMedia }` which seems to be more generally usable. We can change later if it should become necessary.

Reply:
Wouldn't it be better to do 
```
const isMedia = ['image', 'video'].includes( selectedElement.type );
```

Reply:
@miina Yeah. Makes sense. I like `isMedia`. It's better for now to add more features w.r.t. how the image/video/etc work.